File: trunk/Java/OPSJLib/src/ops/FilterQoSPolicy.java
/*
 * FilterQoSPolicy.java
 *
 * Created on den 10 oktober 2007, 10:30
 *
 * To change this template, choose Tools | Template Manager
 * and open the template in the editor.
 */

package ops;

/**
 *
 * @author angr
 */
public interface FilterQoSPolicy
{
    
    /** Creates a new instance of FilterQoSPolicy */
    public boolean applyFilter(OPSObject o);
    
}

File: trunk/C#/Ops/FilterQoSPolicy.cs
///////////////////////////////////////////////////////////
//  FilterQoSPolicy.cs
//  Implementation of the Interface FilterQoSPolicy
//  Created on:      12-nov-2011 09:25:30
//  Author:
///////////////////////////////////////////////////////////

namespace Ops 
{
	public interface IFilterQoSPolicy 
    {
		/// 
		/// <param name="o"></param>
		bool ApplyFilter(OPSObject o);
	}

}